### PR TITLE
Cozy

### PR DESCRIPTION
**Pull request by MostAwesomeDude**
###### Motivation for this change

I want to play with [Cozy](https://github.com/CozySynthesizer/cozy). I needed #42603.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/libraries/igraph/default.nix [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  buildInputs = [ flex yacc zlib libxml2 ];
+
+  # This file is normally generated by igraph's bootstrap.sh, but we can do it
+  # ourselves. ~ C.
+  postPatch = ''
+    echo "${version}" > VERSION
+  '';
+
+  meta = {
+    homepage = http://igraph.org/;
```
Comment (fpletz) [inline pkgs/development/libraries/igraph/default.nix]: Please add a description.

File: pkgs/development/python-modules/dictionaries/default.nix [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  version = "0.0.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1jx2ph509sk4l7spslz16y8l6xn97d13nspn4ds2lxn5ward9ihy";
+  };
+
+  meta = {
+    description = "Dict implementations with attribute access";
+    maintainers = [ lib.maintainers.MostAwesomeDude ];
```
Comment (fpletz) [inline pkgs/development/python-modules/dictionaries/default.nix]: Please add the license.

File: pkgs/development/python-modules/python-igraph/default.nix [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  buildInputs = [ pkgconfig igraph ];
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0xp61zz710qlzhmzbfr65d5flvsi8zf2xy78s6rsszh719wl5sm5";
+  };
+
+  meta = {
+    description = "High performance graph data structures and algorithms";
+    maintainers = [ lib.maintainers.MostAwesomeDude ];
```
Comment (fpletz) [inline pkgs/development/python-modules/python-igraph/default.nix]: Please add the license.

**Review by FRidh — CHANGES_REQUESTED** [review]
Split f6f2b38ec9e04c2a0efc778b4fec39b9fef2971c over the commits that you modified.

**Comment by FRidh** [comment]
@GrahamcOfBorg build python.pkgs.dictionaries python3.pkgs.dictionaries python.pkgs.python-igraph python3.pkgs.python-igraph

**Comment by MostAwesomeDude** [comment]
FWIW I just want to indicate that I am still alive on this branch and ready for re-review at any time.

**Comment by Ekleog** [comment]
(triage) I think this is ready to merge?

**Pull request merged**